### PR TITLE
[#60] Dropped grpc keepalive options

### LINKED ISSUE #60: Drop gRPC tuning options, support TLS connections
Remove keepalive options, try specifying TLS-enabled addresses (they should work out of the box) and update documentation. Blocked by nspcc-dev/neofs-sdk-go#9.

### PR DESCRIPTION
**Pull request by KirillovDenis**
closes #60

Signed-off-by: Denis Kirillov <denis@nspcc.ru>

### CODE REVIEW

**Review by roman-khimov — COMMENTED** [review]
Have you tried it with grpcs?

File: README.md [hunk 1]
```diff
@@ -57,6 +57,11 @@ backend node (and otherwise default settings):
 $ neofs-http-gw -p 192.168.130.72:8080
 $ HTTP_GW_PEERS_0_ADDRESS=192.168.130.72:8080 neofs-http-gw
 ```
+It's also possible specify uri scheme (grpc or grpcs) when using `-p`:
```
Comment (roman-khimov) [inline README.md:60]: possible to

**Comment by KirillovDenis** [comment]
> Have you tried it with grpcs?

Yes I tried

**Comment by roman-khimov** [comment]
Squash, please.

**Review by roman-khimov — APPROVED** [review]

**Pull request merged**